automated-testing/unit-testing/readme.md

```diff
@@ -2,7 +2,7 @@
 
 ## Goals
```

Comment:
This looks to be an unintentional role back of one of the changes in Kanio's previous PR for testing. The removed red text "_Unit tests play an integral role in building quality software and enabling agile methodologies. The implementation of the unit is finished when the unit satisfies the tests._" should be the text displayed (until the unit testing readme itself is rewritten).

Reply:
replaced readme.md for the Unit tests from the copy from Master branched